test/functional/wallet_bumpfee.py

```diff
@@ -279,6 +281,62 @@ def test_maxtxfee_fails(test, rbf_node, dest_address):
     test.restart_node(1, test.extra_args[1])
     rbf_node.walletpassphrase(WALLET_PASSPHRASE, WALLET_PASSPHRASE_TIMEOUT)
 
+def test_watchonly_psbt(test, rbf_node, dest_address):
+    rbf_wallet_rpc = rbf_node.get_wallet_rpc("")
+    rbf_node.createwallet(wallet_name="watchonly", disable_private_keys=True, blank=True) # empty, non-privkey wallet
+    watchonly_endpoint = rbf_node.get_wallet_rpc("watchonly")
+
+    # We need all inputs to be "from" the watchonly wallet from the upcoming spend
+    # so don't send it off quite yet
+    tx_to_bump = spend_one_input(node=rbf_wallet_rpc, dest_address=dest_address, send=False)
+    decoded_tx = rbf_node.decoderawtransaction(tx_to_bump)
+    assert_equal(len(decoded_tx["vin"]), 1)
+    assert_equal(len(decoded_tx["vout"]), 2)
+    tx_in = decoded_tx["vin"][0]
+    prevtx = rbf_wallet_rpc.gettransaction(tx_in["txid"], False, True)
+    prev_out_addr = prevtx["decoded"]["vout"][tx_in["vout"]]["scriptPubKey"]["addresses"][0]
+    prev_addr_info = rbf_wallet_rpc.getaddressinfo(prev_out_addr)
+    change_addr_info = None
+    for output in decoded_tx["vout"]:
+        output_addr = output["scriptPubKey"]["addresses"][0]
+        output_info = rbf_wallet_rpc.getaddressinfo(output_addr)
+        if output_info["ischange"]:
+            change_addr_info = output_info
+
+    assert change_addr_info is not None
+
+    # Import spend_one_input's change address as change address so value can be reduced
+    # during bump
+    watchonly_endpoint.importmulti([{"desc":change_addr_info["desc"], "internal":True, "timestamp":"now", "watchonly":True}])
+    # We have to own previous transaction/input
+    watchonly_endpoint.importmulti([{"desc":prev_addr_info["desc"], "timestamp":"now", "watchonly":True}])
+    # Input tx needs to be in wallet to dummy-sign/estimate size
+    watchonly_endpoint.importprunedfunds(prevtx["hex"], rbf_wallet_rpc.gettxoutproof([prevtx["txid"]]))
+    # We also need change address in keypool, import *something*
+    watchonly_endpoint.importmulti([{"desc":"wpkh(02fcba7ecf41bc7e1be4ee122d9d22e3333671eb0a3a87b5cdf099d59874e1940f)#442qn8e8", "internal":True, "timestamp":"now", "watchonly":True, "keypool":True}])
+
+    # Now that we're watching, shoot off the original tx
+    original_txid = rbf_node.sendrawtransaction(tx_to_bump)
+    assert original_txid in rbf_node.getrawmempool()
+
+    # And replace it once
+    bumped_psbt = watchonly_endpoint.bumpfee(original_txid)
+    assert "psbt" in bumped_psbt
+    assert "txid" not in bumped_psbt
```

Comment:
If you add `assert_greater_than(bumped_psbt["origfee"], 0)` it will throw `AssertionError: -0.00099000 <= 0`

Reply:
fixed and added the test